packages/app/package.json

```diff
@@ -73,7 +73,6 @@
     "react-bootstrap": "^2.4.0",
     "react-bootstrap-range-slider": "^3.0.8",
     "react-copy-to-clipboard": "^5.1.0",
-    "react-csv": "^2.2.2",
```

Comment:
Hasn't been updated since 2020 and has several open issues with comma delimiters